__tests__/nc-games.test.js

```diff
@@ -30,4 +30,35 @@ describe('GET api/categories', () => {
                 })
             });
     });
-})
\ No newline at end of file
+})
+
+describe.only('GET api/reviews/:review_id', () => {
+    test("returns a review object with properties", () => {
+        return request(app) 
+        .get("/api/reviews/1")
+            .expect(200)
+            .then((res) => {
+                    expect(res.body.review[0]).toEqual(
+                        expect.objectContaining({
+                            review_id: expect.any(Number),
```

Comment:
Since we're looking at a specific review and we have the test data, we want to see an object containing those specific details - ie. the review_id would be 1, and the title would be the right title for review_id 1. Thinking of this outside of code - if you went on a website and clicked an article link, you'd want to see that article, not just "any" article right? :)